test/e2e/e2e.test.ts

```diff
@@ -0,0 +1,87 @@
+import { expect } from 'chai';
+import path from 'path';
+import { getFiles, existsMustBeDir } from '../../src/utils';
+import { setupRecording, testNameToWorkingDirectory } from './e2e-utils';
+import { dump, deploy } from '../../src';
+
+const shouldUseRecordings = process.env['AUTH0_HTTP_RECORDINGS'] === 'on';
+const AUTH0_DOMAIN = shouldUseRecordings
```

Comment:
The value of `AUTH0_HTTP_RECORDINGS` env var dictates wether the recordings are used or not. And if not, set the `AUTH0_DOMAIN` to the fake `deploy-cli-dev.eu.auth0.com` domain.